pkg/daemon/daemon.go

```diff
@@ -1079,3 +1088,42 @@ func tryCreateNMUdevRule() error {
 	}
 	return nil
 }
+
+func trySetOvsHWOL(newState bool) (bool, bool, error) {
```

Comment:
newState is too vague. Can we come up with a bbetter name?

Reply:
Updated to "switchdevDevExists"